### DISCUSSION TITLE
Markdown is not recognized anymore if <span> is at the beginning of a paragraph.

### DISCUSSION THREAD

**Discussion by janmaltel** [mdx-js/mdx#1461] [Q&A] [answered]
I want to start some paragraphs with a couple of words in a special style. 
However, if I start the paragraph (in an `.mdx`-file) with a `<span>`-tag, then the remaining text in that paragraph doesn't seem to understand markdown anymore (such as underscores for emphasizing, see following example):

The code block
```
Some text with _some words in italics_.

<span className='someClass'> If the p starts with a span</span>, the _underscores
don't seem to work anymore_.
```

produces

<img width="800" alt="image" src="https://user-images.githubusercontent.com/3419333/107570251-8fdb5480-6be9-11eb-97bb-2c9e27bd62b0.png">

Is this intended? Is there a simple workaround?

Thanks! mdx is awesome :)

**Comment by lachlanjc** [comment]
Have this issue too but couldn't find a good solution, so just wrote lines that had to start with JSX (like an SVG icon component) using HTML for formatting. Would love to know if there's any workaround though.

**Comment by ChristianMurphy** [comment, marked as answer]
MDX 1 does not support this.
MDX 2 (currently in beta) supports this as a part of https://github.com/mdx-js/mdx/issues/628
You can find out more about version 2 and how to try out the beta at https://github.com/mdx-js/mdx/issues/1041

**Comment by janmaltel** [comment]
While MDX 2 seems to indeed support my original issue, it unfortunately breaks my inline math.

**Edit:** Actually it does work now almost perfectly... except for a small bug outlined [here](https://github.com/mdx-js/mdx/issues/1462).
